### ISSUE TITLE
On Device Activity not working?

### ISSUE THREAD

**Issue by expertsonline** [cakebake/node-red-contrib-alexa-remote-cakebaked#39]
Just checking if others are having similar issues? It has stopped working for me for the last few hours. I mainly use this to see which device /room the event is triggered from...

**Comment by jasazick** [comment]
Also having that same issue - everything else seems to be working

**Comment by CobraVmax** [comment]
I have the same problem since updating & rebooting

**Comment by St4tikk** [comment]
Same issue here.  I rebooted the box node-red is on after some maintenance and now on device activity is dead.

**Comment by obaldius** [comment]
Me too

**Comment by Cjkeenan** [comment]
https://github.com/Apollon77/alexa-remote/issues/132#issue-1011417647 looks like it is a bigger issue than we thought.

**Comment by vbenhur** [comment]
me too

**Comment by vbenhur** [comment]
Same issue here.

**Comment by mnbahena** [comment]
It also stopped working for me, I've looked it up in several places, but I haven't found anything yet :(

**Comment by Cjkeenan** [comment]
Working for me again. Make sure you refresh your cookie and restart all of your flows.

**Comment by obaldius** [comment]
It's been coming and going over the last few days. Let's see what happens when it settles

**Comment by drvirtue** [comment]
Same here. Stopped working about a week ago. Haven't found a workaround that is successful for me yet.

**Comment by Cjkeenan** [comment]
Still working for me (US). Did you try manually refreshing your cookie?

**Comment by drvirtue** [comment]
Ah - I did not realize there was a difference between just reinitializing (from the Proxy), and dropping/recreating the proxy file. 
 I did that, and now that I have reestablished my Amazon logon/pw, the Device Activity node appears to be working... sometimes.  I'm not convinced it is consistently working like it was.  But at least now I have had some successes.  I will continue to monitor.  Thanks for the insights.  Hopefully a permanent fix will resolve the underlying issue.

**Comment by pmendiuk** [comment]
It's working intermittently this morning.  It seems to work once and then become unresponsive for some period of time.  There is a significant delay between the Alexa routine firing and the Device Activity response if it fires.

**Comment by iceFbr** [comment]
Refresh the cookies did not worked for me...

**Comment by pmendiuk** [comment]
@Cjkeenan Is it still working for you?  It was working this morning until I redeployed the flow.

**Comment by Cjkeenan** [comment]
Yes it is, but I have not redeployed or refreshed my cookie since the outage, so I rather not test my luck. I hope you can understand 😄

**Comment by drvirtue** [comment]
Mine worked for a few days after a cookie refresh.  But, now it has stopped hearing Device Activity again - even after refreshing.

**Comment by expertsonline** [comment]
Same ... been down for me as well.

**Comment by Cjkeenan** [comment]
Down for me now as well. Not a great couple weeks for big tech...

**Comment by hannoverRed** [comment]
can it be that it doesn't work again?

**Comment by obaldius** [comment]
Down again and for a long time. It seems it has settled

**Comment by Apollon77** [comment]
New version available @cakebake see https://github.com/Apollon77/alexa-remote/issues/132#issuecomment-939825864 for more infos. 

You CAN NOT simply copy over files because of the needed changes

**Comment by obaldius** [comment]
Hi @cakebake , are you planning on updating your fork? Many thanks for everyhing you've done..

**Comment by Varguit** [comment]
Migrating to Appelstrudel fork as @cakebake hasn't updated and my whole home automation depends on device activity functionality
 [https://github.com/bbindreiter/node-red-contrib-alexa-remote2-applestrudel](url)

**Comment by obaldius** [comment]
> Migrating to Appelstrudel fork as @cakebake hasn't updated and my whole home automation depends on device activity functionality [https://github.com/bbindreiter/node-red-contrib-alexa-remote2-applestrudel](url)

Mine has come back to live again but who knows for how long. Anyways, can you please share how you migrated please? thanks in advanced

**Comment by Cjkeenan** [comment]
There are a couple of ways to migrate, but this is by far the easiest and it does not disturb your flows: https://tech.scargill.net/node-red-contrib-alexa-cakebaked/

**Comment by obaldius** [comment]
Surprisingly, it's been working for 8 days now.

**Comment by sicampbell82** [comment]
Has anyone else started having issues again, mine stopped working some time today after working fine since the fix

**Comment by obaldius** [comment]
@sicampbell82 yep, it crapped out today. What fix did you apply?

**Comment by sicampbell82** [comment]
I haven't found any yet, restarted everything and all Auth/cookies seem fine

**Comment by Cjkeenan** [comment]
I had issues as well, but I forced a cookie refresh and it started working again. US and using applestrudel

**Comment by sicampbell82** [comment]
Did not work for me.  Thanks

**Comment by bwims** [comment]
I just migrated to AppleStrudel and it works fine there.   I have used it before so I have to ask - what do people use it for?

Incidentally for those of us who use Dietpi, the location of the node red modules is here...

> /mnt/dietpi_userdata/node-red/node_modules

and the way to migrate is as Pete Scargill says:

> npm remove node-red-contrib-alexa-remote
> npm remove node-red-contrib-alexa-cakebaked
> npm install node-red-contrib-alexa-remote2-applestrudel
> dietpi-services restart node-red

B.

**Comment by obaldius** [comment]
> and the way to migrate is as Pete Scargill says:
> 
> > npm remove node-red-contrib-alexa-remote
> > npm remove node-red-contrib-alexa-cakebaked
> > npm install node-red-contrib-alexa-remote2-applestrudel
> > dietpi-services restart node-red
> 
> B.

Sorry for the newbie question but, regarding this npm install thing. Can this be done whithin a Hassio install? How? Terminal & SSH maybe?

thanks in advanced

**Comment by bwims** [comment]
Can't help there, don't use Hassio, sorry.

**Comment by elgerg** [comment]
@cakebake is this repo still being maintained? IE are you going to update it or should we switch to AppleStrudel?

Thanks for the work.

**Comment by obaldius** [comment]
> @cakebake is this repo still being maintained? IE are you going to update it or should we switch to AppleStrudel?
> 
> Thanks for the work.

I made the same question a couple of weeks ago and got no answer... Yesterday I moved to applestrudel.

**Comment by elgerg** [comment]
@obaldius I guess you are right.. Time to move...
